pages/auth.tsx

```diff
@@ -138,7 +160,17 @@ const Authentication = () => {
         {/* step two: image input */}
         <div className={pageNo === 3 ? styles.form : global.hidden}>
           <h1>add image</h1>
-          <input type="image" name="name" id="name"></input>
+          {/* display string base64 for url as an image and fit image with good resolution */}
+          <img src={preview} style={{objectFit:"contain"}} width='50%' height='50%' />
+              {/* get url of image when it is selected and/or changed */}
+              <input type={"file"} accept="image/*" onChange={(event)=>{ const file = event.target.files![0]
+              if (File){
+                setImage(file);
+              }else{
+                error;
```

Comment:
the error function should be an await error(code). We need to add the error code for this particular instance in the error file with a code. @catastrxphic